dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/utils/ProxySupportUtil.java

```diff
@@ -33,13 +36,45 @@ public class ProxySupportUtil {
     private static URL extractURL(HttpServletRequest request) {
         String urlString;
         String refererHeader = request.getHeader(HTTP_HEADER_REFERER);
+
+        String docPath = "/openapi";
+        String uiPath = docPath+"/ui";
```

Comment:
- We should have constants somewhere for the default paths
- Code has not been formatted.